src/main/java/gov/cms/mat/cql_elm_translation/controllers/CqlConversionController.java

```diff
@@ -77,7 +76,7 @@ public CqlConversionPayload cqlToElmJson(
             .build();
     cqlLibraryService.setUpLibrarySourceProvider(cqlData, accessToken);
 
-    return cqlConversionService.translateCqlToElm(requestData, checkContext);
+    return cqlConversionService.translateCqlToElm(requestData);
```

Comment:
is this not required anymore? this might cause an issue for libraries.

Reply:
Reverted